test/test_util/get_patch_context.py

```diff
@@ -15,6 +15,16 @@ def get_add_patch_context(
 
     return root.addBlockHeader(offset).locationToken()
 
+def get_replace_patch_context(
+    file: Path,
+    offset: int,
+) -> PatchfileParser.LocationTokenContext:
+    root = CommonParseManager(PatchfileLexer, PatchfileParser).get_root(
+        file
+    ).replaceNthBlock(0)
```

Comment:
Consider storing the CommonParseManager in a variable before using it, since this line is quite difficult to read and would ideally be broken up.

Reply:
Sounds good, I'll update our code with this!